protocol/protobuf/pairing.proto

```diff
@@ -91,8 +91,9 @@ message SyncPairInstallation {
   string installation_id = 2;
   string device_type = 3;
   string name = 4;
-  // following fields used for local pairing
+  // used for local pairing
   uint32 version = 5;
+  string target_installation_id = 6;
```

Comment:
Actually, it's weird that we have this here. I'd rather `SyncInstallation` message and a `PairInstallation` that would compose the former one. But it might be too difficult to refactor now I guess.